dart/tool/service.md

```diff
@@ -2068,11 +2094,6 @@ class @Isolate extends Response {
   // The id which is passed to the getIsolate RPC to load this isolate.
   string id;
 
-  // Provided and set to true if the id of an Object is fixed. If true, the id
```

Comment:
@bkonyi - just want to confirm that this removal in service.md is intentional?

Reply:
It looks like the VM still returns `IsolateRef`s that contain `fixedId` fields. Should that field be removed from the VM response? Or, should IsolateRef be changed to subclass `ObjRef` (it currently subclasses `Response`)?

Reply:
From an off-line conversation, I'd added this field in as a patch to the service.md file a few months ago; we'll investigate whether this or something like it should be upstreamed.